docs-chef-io/content/inspec/resources/ipfilter.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "ipfilter resource"
+draft = false
+gh_repo = "inspec"
+platform = "bsd"
+platform = "solaris"
```

Comment:
A couple issues here:

1. This platform parameter is used to filter resource pages in this page: https://docs.chef.io/inspec/resources/ and we don't currently have bsd or solaris platforms listed there (although we could add them)

2. The toml spec doesn't accept repeating entries. 

It's possible that we could define this as a list of platforms, but I don't think that will work on the resource list page right now.

So I guess I have two questions: 
1. Would this page fit under one of the existing headings on the inspec/resource page? (I'm guessing not)
2. Will there be a bunch more resources that are in bsd and/or solaris? In which case I can add headings for both and figure out how to handle these platforms in a list.

Reply:
@ahasunos - please clarify the questions raised by Ian.

Reply:
There will be several more resources that are bsd/ solaris only. As bsd is more popular than solaris, if we're only allowed one tag, I'd go with bsd.